test/e2e/storage/drivers/csi.go

```diff
@@ -197,14 +190,8 @@ func (g *gcePDCSIDriver) GetDriverInfo() *testsuites.DriverInfo {
 	return &g.driverInfo
 }
 
-func (g *gcePDCSIDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
```

Comment:
curious as to the reason why we're removing `SkipUnsupportedTest` and moving the skip logic into `CreateDriver()`?

Reply:
This is because this PR is based on https://github.com/kubernetes/kubernetes/pull/70992. When looking at overall changed files, you'll also see changes from commits in that PR, like this one.

Better have a look at the recent commits that are exclusive to this PR ("introduce TestVolume interface", "use local TestConfig, remove indirection").

Sorry for the confusion. I have multiple PRs pending right now that all conflict with each other, therefore I am working on a single branch :-/

Reply:
Ah I see, I will look at specific commits for future review